packages/web/src/components/organisms/PropertyCardList/CurrencySwitcher.tsx

```diff
@@ -28,25 +27,28 @@ const Circle = styled.div<{ isActive?: boolean }>`
 
 const CurrencyContainer = styled.div`
   display: flex;
-  justify-content: stretch;
+  justify-content: flex-end;
   align-items: center;
+  margin-left: 30px;
 `
 
-export const CurrencySwitcher = ({ className }: Props) => {
```

Comment:
We cannot remove the className option.

Reply:
fix in df0e033790a2de3bc1d4a8ccdb582725d469f1c9